library/options.inc.php

```diff
@@ -1342,6 +1342,20 @@ function generate_form_field($frow, $currvalue)
         echo "<img class='signature' id='form_{$field_id_esc}_img' title='$description'
             data-pid='$cpid' data-user='$cuser' data-type='$datatype'
             data-action='fetch_signature' alt='Get Signature' src='" . attr($currvalue) . "'>\n";
+    } elseif ($data_type == 43) {
+        echo generate_select_list(
+            "form_$field_id",
+            $list_id,
+            $currvalue,
+            $description,
+            $showEmpty ? $empty_title : '',
+            "select-dropdown",
+            $lbfchange,
+            '',
+            null,
```

Comment:
also need to incorporate `$disabled` in above (like is done in data_type 1)

Reply:
```suggestion 
            $disabled,
```